src/components/localization/language-dropdown.tsx

```diff
@@ -158,16 +153,11 @@ const LanguageText = styled(Text)`
 
 /* stylelint-enable */
```

Comment:
do we need this line in this file?

Reply:
Nope most likely, just a comment someone else had left